Tasks/Liaskovski/Task2/task2.py

```diff
@@ -0,0 +1,41 @@
+import datetime 
+Current_time  = datetime.datetime.now().strftime('%H:%M') 
+
+def answer(hours, min):
+   text=["один", "два", "три", "четыре", "пять(ого)", "шесть(ого)",
+    "семь", "восемь","девять(ого)","десять(ого)", "одинадцать", "двенадцать", "тринадцать", "четырнадцать", "пятнадцать", "шеснадцать",
+    "семнадцать", "восемнадцать", "девятнадцать", "двадцать(ого)", "двадцать один","двадцать два", "двадцать три", "двадцать четыре", "двадцать пять",
+    "двадцать шесть", "двадцать семь", "двадцать восемь", "двадцать девять", "половина","полночь"]
```

Comment:
kekekekeke, it's creative, but it goes against the task description, -15 points for the logical mistake